src/home/rendering/display-github-user-information.ts

```diff
@@ -11,7 +11,13 @@ export async function displayGitHubUserInformation(gitHubUser: GitHubUser) {
   if (!toolbar) throw new Error("toolbar not found");
 
   const img = document.createElement("img");
-  img.src = gitHubUser.avatar_url;
+  if (gitHubUser.avatar_url) {
+    img.src = gitHubUser.avatar_url;
+  } else {
+    img.style.width = "50px";
```

Comment:
Style is to be handled by CSS. Also are you sure it renders at 50px? I would assume 48px.